images/sds-local-volume-scheduler-extender/src/pkg/cache/cache.go

```diff
@@ -43,18 +47,56 @@ type pvcCache struct {
 }
 
 // NewCache initialize new cache.
-func NewCache(logger logger.Logger) *Cache {
-	return &Cache{
-		log: logger,
+func NewCache(logger logger.Logger, pvcExpDurSec time.Duration) *Cache {
+	ch := &Cache{
+		log:             logger,
+		expiredDuration: pvcExpDurSec,
 	}
+
+	go func() {
+		timer := time.NewTimer(ch.expiredDuration)
+
+		for range timer.C {
+			ch.clearBoundExpiredPVC()
+			timer.Reset(ch.expiredDuration)
```

Comment:
This reset will make it work twice less frequent then specified in `pvcExpDurSec`. For example, if it's 10 minutes, timer will be called once in 20 minutes. This is could be misleading during test.

Reply:
Oops, sorry, I'm wrong.

Reply:
Not really. 
`I0926 15:50:09.872456       1 cache.go:69] "DEBUG [clearBoundExpiredPVC] starts to clear expired PVC"
I0926 15:50:09.872498       1 cache.go:93] "DEBUG [clearBoundExpiredPVC] finished the expired PVC clearing"
I0926 15:50:39.873456       1 cache.go:69] "DEBUG [clearBoundExpiredPVC] starts to clear expired PVC"
I0926 15:50:39.873483       1 cache.go:93] "DEBUG [clearBoundExpiredPVC] finished the expired PVC clearing"
I0926 15:51:09.874074       1 cache.go:69] "DEBUG [clearBoundExpiredPVC] starts to clear expired PVC"
I0926 15:51:09.874109       1 cache.go:93] "DEBUG [clearBoundExpiredPVC] finished the expired PVC clearing"
`